docs/dev_cycle.part-02.rst

```diff
@@ -0,0 +1,174 @@
+Part 2: Compilation and Tester Deployment
+=========================================
+
+.. contents:: :local:
+
+Compile
+-------
+
+Solidity compiles the source to EVM bytecode, the operations codes that will actually
+run on the blockchain.
+
+To compile, first check that solidity works on your machine. Try to get the latest version.
```

Comment:
This might be a good place to drop in a snippet about using `py-solc` to install a specific version of `solc`